live-examples/webapi-examples/url/url.js

```diff
@@ -1,9 +1,9 @@
-const url = new URL('https://en.wikipedia.org/wiki/Mozilla#Software');
+const url = new URL('https://example.com/wiki/Mozilla#Software');
 console.log(url.protocol);
 // expected output: "https:"
 
 console.log(url.hostname);
-// expected output: "en.wikipedia.org"
+// expected output: "example.com"
```

Comment:
here, the original is more informative than the edit.

Reply:
@estelle @bsmth It perfectly illustrates its api with an example domain.